CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # CHANGELOG
 
 ## XX.XX.XX - 20XX-XX-XX
+
+## 20.43.1 - 2024-05-20
```

Comment:
We should change this to be `20.44.0` we introduced a big new feature with lots of new public APIs!

Reply:
ack, reverting this one